lock.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+# This should be the base image the Docker image built by the GitHub Action uses
+BUILD_IMAGE="mambaorg/micromamba:1.4.9-bullseye-slim"
+docker pull "${BUILD_IMAGE}"
+
+# Don't need to ensure emulation possible on non-x86_64 platforms at the
+# `docker run` level as the platform is specified in the conda-lock command.
+docker run \
+    --rm \
+    --volume "${PWD}":/work \
+    --workdir /work \
+    "${BUILD_IMAGE}" \
+    /bin/bash -c "\
+        micromamba install --yes --channel conda-forge conda-lock && \
+        conda-lock lock --micromamba --platform linux-64 --file environment.yml"
```

Comment:
For maintainability I would like to know if this Bash script is clear enough as to what is happening and why (hopefully the name `lock.sh` helps it a bit as well).